preflight_checks.yml

```
@@ -0,0 +1,314 @@
+---
+- name: Initialize preflight results list
+  set_fact:
+    preflight_results: []
+    preflight_failures: []
+
+- name: Gather all Ansible facts
+  setup:
+
+- name: Check if OS is RHEL 9+
+  set_fact:
```

please, use FQCN everywhere:
```suggestion
  ansible.builtin.set_fact:
```

---

Updated with FQCN.